doc/how-to/getting_started_tcm.rst

```diff
@@ -0,0 +1,387 @@
+#.  Log in with the ``admin`` username. The initial password is generated automatically
+    and printed in the TCM log in a message like this:
+
+    ..  code-block:: text
+
+        Jan 24 05:51:28.443 WRN Generated super admin credentials login=admin password=qF3A5rjGurjAwmlYccJ7JrL5XqjbIHY6
+
+    Paste this password into the login form and click **Log in**.
+
+After a successful login, you see the |tcm| web UI:
+
+.. image:: images/tcm_start_empty_cluster.png
+    :width: 700
+    :align: center
+    :alt: TCM stateboard with empty cluster
+
+..  _getting_started_tcm_cluster:
+
+Setting up a Tarantool EE cluster
+---------------------------------
+
+To prepare a Tarantool EE cluster, complete the following steps:
+
+#.  Configure the cluster in |tcm|. |tcm| provides a web-based editor for writing
+    cluster configurations. It is connected to the configuration storage (etcd in
+    this case): all changes you make in the browser are sent to etcd in one click.
+#.  Start the cluster instances locally using the :ref:`tt <tt-cli>` utility.
+
+..  _getting_started_tcm_cluster_config:
+
+Configuring a cluster in TCM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A freshly installed |TCM| has a predefined cluster named **Default cluster**. It
+doesn't have any configuration or topology out of the box. Its initial properties
+include the default etcd and Tarantool connection parameters. Check these properties
+to find out where TCM sends the cluster configuration that you write.
+
+To view the **Default cluster**'s properties:
+
+#.  Go to **Clusters** and click **Edit** in the **Actions** menu opposite the cluster name.
+
+    .. image:: images/tcm_start_cluster_edit.png
+        :width: 700
+        :align: center
+        :alt: TCM edit cluster
+
+#.  Click **Next** on the **General** tab.
+
+    .. image:: images/tcm_start_cluster_general.png
+        :width: 700
+        :align: center
+        :alt: General cluster settings
+
+#.  Find the connection properties of the configuration storage that the cluster uses.
+    By default, it's an etcd storage with a prefix ``/default`` running on port
+    ``2379`` (default etcd port) on the same host. Click **Next**.
+
+    .. image:: images/tcm_start_cluster_storage.png
+        :width: 700
+        :align: center
+        :alt: Cluster configuration storage settings
+
+#.  Check the Tarantool user that |tcm| uses to connect to the cluster instances.
+    It's ``guest`` by default. Click **Update** to save the changes and return to
+    the **Clusters** page.
+
+    .. image:: images/tcm_start_cluster_tarantool.png
+        :width: 700
+        :align: center
+        :alt: Cluster Tarantool connection settings
+
+Next, write the cluster configuration and upload it to the etcd storage:
+
+#.  Go to **Configuration**.
+#.  Click **+** and provide an arbitrary name for the configuration, for example, ``all``.
+#.  Paste the following YAML configuration into the editor:
```

Comment:
> YAML configuration

Maybe, add a link to https://www.tarantool.io/en/doc/latest/concepts/configuration/#configuration-overview ?